docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1305 @@
+{
```

Comment:
<p>Probably good to add a brief comment why the 0 replacement. I know but might be puzzling when reading this the first time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Good point, I added some more text to that section in general now.